src/Database/Schema/Table.php

```diff
@@ -82,6 +82,38 @@ class Table
     protected $_temporary = false;
 
     /**
+     * Column length when using a `tiny` text column type
+     *
+     * @var int
+     */
+    const LENGTH_TINY = 255;
+
+    /**
+     * Column length when using a `medium` text column type
+     *
+     * @var int
+     */
+    const LENGTH_MEDIUM = 16777215;
+
+    /**
+     * Column length when using a `long` text column type
+     *
+     * @var int
+     */
+    const LENGTH_LONG = 4294967295;
+
+    /**
+     * Valid column length that can be used with text type columns
+     *
+     * @var array
+     */
+    public static $columnLengths = [
+        'tiny' => self::LENGTH_TINY,
+        'medium' => self::LENGTH_MEDIUM,
+        'long' => self::LENGTH_LONG
+    ];
```

Comment:
I added this in the Table class because there are some places specific to other DB vendors where a test on these values are needed.
It felt weird to have mention about Mysql in other DB vendors.